api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -44,7 +44,7 @@ private class PurchasesAPI {
         activity: Activity,
         storeProduct: StoreProduct,
         packageToPurchase: Package,
-        purchaseOption: PurchaseOption,
+        purchaseOption: PurchaseOption?,
```

Comment:
I also considered separating the purchase methods for OTP and subscriptions but I didn't like that much...